### PR TITLE
Add Button and ButtonGroup

### PR DESCRIPTION
**Pull request by marthayaputra**

### CODE REVIEW

File: src/General/@next/Button/ButtonStyle.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+const buttonSizeStyleMap: Record<ButtonSize, string> = {
+  default: defaultButtonSizeStyle,
+  slim: slimButtonSizeStyle,
+  large: largeButtonSizeStyle,
+};
+
+export const BasicButton = styled.button.attrs(
+  ({ active, fullWidth }: ButtonProps) => {
+    const activeStyle = active
```
Comment (ninariccimarie) [inline src/General/@next/Button/ButtonStyle.ts]: you can remove the ternary operators and return style as:

```ts
{
  style: {
    ...(active && activeStyle),
    ...(fullWidth && widthStyle)
  }
}
```

**Review by ninariccimarie — COMMENTED** [review]
I think you missed exporting from `src/index.ts`?

File: src/General/@next/Button/ButtonStyle.ts (hunk 1 above)
Comment (marthayaputra) [reply src/General/@next/Button/ButtonStyle.ts]: That looks better, thanks!

**Review by ninariccimarie — APPROVED** [review]
LGTM 🚀

**Pull request merged**